### ISSUE TITLE
Is Message IPC Module Code revert with error ？

### ISSUE THREAD

**Issue by yiliang114** [eclipse-theia/theia#11618]
### Steps to Reproduce:

1.  Get the latest code from master,  now the latest submitted hash value is [da356b5](https://github.com/eclipse-theia/theia/commit/da356b537037680cdc7283a695ebfbf0c00d2cf1)
2. Try to run `examples/browser`, execute `npm run start`command in `theia/examples/browser` after building theia packages.
3. The application cannot start successfully, It has been in loading state and the browser console has not reported any errors. 

No more log output from browser console.
![image](https://user-images.githubusercontent.com/11473889/187442367-947de123-e513-44fd-b035-06056a8df198.png)

![image](https://user-images.githubusercontent.com/11473889/187442164-e4424eef-f40f-4ae2-a23d-9e24731976d7.png)

```
2022-08-30T12:55:40.821Z root ERROR TypeError [ERR_INVALID_SYNC_FORK_INPUT]: Asynchronous forks do not support Buffer, TypedArray, DataView or string input: 'overlapped'
    at internal/child_process.js:1025:15
    at Array.reduce (<anonymous>)
    at getValidStdio (internal/child_process.js:956:17)
    at ChildProcess.spawn (internal/child_process.js:343:11)
    at spawn (child_process.js:667:9)
    at spawnWithSignal (child_process.js:871:17)
    at Object.fork (child_process.js:153:10)
    at IPCConnectionProvider.fork (/Users/jinjing/projects/yiliang114/fe/ide/theia/packages/core/lib/node/messaging/ipc-connection-provider.js:81:33)
    at IPCConnectionProvider.doListen (/Users/jinjing/projects/yiliang114/fe/ide/theia/packages/core/lib/node/messaging/ipc-connection-provider.js:40:35)
    at IPCConnectionProvider.listen (/Users/jinjing/projects/yiliang114/fe/ide/theia/packages/core/lib/node/messaging/ipc-connection-provider.js:37:21)
```

I was able to successfully start and develop it not long ago, this is my strange point. So I look for code changes in the `Message/IPC-xxx` modules. Found the commit that may have problems is [73260a1c](https://github.com/eclipse-theia/theia/commit/73260a1c5e8b5a4237f39588d36cc382b868cfc9),  related to #11561.  

So I revert the code on my local and retest it, It's normal again

after I revert code:  

![image](https://user-images.githubusercontent.com/11473889/187445572-8a0aac94-5337-4efd-a8d1-0e973e40b7af.png)

![image](https://user-images.githubusercontent.com/11473889/187441169-66d7f808-7877-40cd-9a7e-56e1d2d990bd.png)

### Additional Information

- Operating System:  Macos 
- Theia Version:  da356b537037
- Node Version: v14.17.6

**Comment by yiliang114** [comment]
@paul-marechal Excuse me, I'm not sure about this problem, can you take up your time to help me look at it?

**Comment by msujew** [comment]
@yiliang114 The `overlapped` argument is only available starting with node [`14.18.0`](https://nodejs.org/ca/blog/release/v14.18.0/) Do you mind updating your node version and testing it again?

**Comment by yiliang114** [comment]
@msujew Sure, I originally tested it on a later version, but it seems that the operation failed due to cache dependency or other problems.  Anyway, now that I've cleared the cache, it's working properly. But If since the lowest version is, do we need to mark it in package.json?

```json
  "engines": {
    "yarn": ">=1.7.0 <2",
    "node": ">=14.18.0"
  },
```
